### PR TITLE
refactor: migrate remaining v-list

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/NavigationMenu.vue [hunk 1]
```diff
@@ -219,146 +219,64 @@ const navItems: MenuItem[] = [
...
-            :to="navItem.route"
+  <div class="p-2" :class="{ '!p-0': isMini }">
+    <template v-for="(navItem, i) in navItems">
+      <RouterLink
+        v-if="navItem.type === 'item'"
+        :key="i"
+        #default="{ isActive, href }"
+        :to="navItem.route"
+      >
+        <a :href="href">
```
Comment (kelsos) [inline frontend/app/src/components/NavigationMenu.vue:230]: do we need both `Routerlink` and `a` here?

Comment (lukicenturi) [reply frontend/app/src/components/NavigationMenu.vue:230]: Yes because I need to use the default template, to get the `isActive`

Comment (kelsos) [reply frontend/app/src/components/NavigationMenu.vue]: Oh ok and the default template if you don't set it has an `a` in the slot

Comment (lukicenturi) [reply frontend/app/src/components/NavigationMenu.vue]: yes exactly

File: frontend/app/src/components/dashboard/VisibleColumnsSelector.vue [hunk 2]
```diff
@@ -49,35 +49,45 @@ const onVisibleColumnsChange = async (visibleColumns: TableColumn[]) => {
...
-    >
-      <template v-for="(item, i) in availableColumns">
-        <VListItem :key="i" :value="item.value">
-          <template #default="{ active }">
-            <VListItemContent>
-              <VListItemTitle>
-                {{ item.text }}
-              </VListItemTitle>
-            </VListItemContent>
+  <div class="py-2">
```
Comment (kelsos) [inline frontend/app/src/components/dashboard/VisibleColumnsSelector.vue:70]: Wasn't removing this vertical padding on the figma page were Raquel adds feedback? 

I will merge as it is but we should probably remove it later.

**Review by kelsos — APPROVED** [review]
lgtm, thanks @lukicenturi

**Pull request merged**